students/skowronski_rafal/lesson_02_flow_control/lesson_06/problem_12.py

```diff
@@ -0,0 +1,24 @@
+# Solves problem 12 - The second maximum
+
+
+def _main():
+    print('Enter sequence of integers:')
+
+    maximum = second_maximum = 0
+    while True:
```

Comment:
You can use while number !=0

Reply:
Yes, I can. But then I have to declare the 'number' variable outside 'while' loop. I just did not want that.